apps/web/screens/Grants/Home/GrantsHome.tsx

```diff
@@ -0,0 +1,202 @@
+import { useIntl } from 'react-intl'
+import NextLink from 'next/link'
+
+import {
+  ArrowLink,
+  Box,
+  Breadcrumbs,
+  CategoryCard,
+  GridColumn,
+  GridContainer,
+  GridRow,
+  Stack,
+  Text,
+} from '@island.is/island-ui/core'
+import { Locale } from '@island.is/shared/types'
+import { GrantSearchSection, GrantWrapper } from '@island.is/web/components'
+import { SLICE_SPACING } from '@island.is/web/constants'
+import {
+  ContentLanguage,
+  CustomPageUniqueIdentifier,
+  GenericTag,
+  Query,
+  QueryGetGenericTagsInTagGroupsArgs,
+} from '@island.is/web/graphql/schema'
+import { useLinkResolver } from '@island.is/web/hooks'
+import { withMainLayout } from '@island.is/web/layouts/main'
+
+import {
+  CustomScreen,
+  withCustomPageWrapper,
+} from '../../CustomPage/CustomPageWrapper'
+import { GET_GENERIC_TAGS_IN_TAG_GROUPS_QUERY } from '../../queries/GenericTag'
+import { m } from '../messages'
+
+const GrantsHomePage: CustomScreen<GrantsHomeProps> = ({
+  categories,
+  locale,
+}) => {
+  const { formatMessage } = useIntl()
+  const { linkResolver } = useLinkResolver()
+
+  const baseUrl = linkResolver('styrkjatorg', [], locale).href
+  const searchUrl = linkResolver('styrkjatorgsearch', [], locale).href
+
+  const breadcrumbItems = [
+    {
+      title: 'Ísland.is',
+      href: linkResolver('homepage', [], locale).href,
+    },
+    {
+      title: formatMessage(m.home.title),
+      href: baseUrl,
+    },
+  ]
+
+  return (
+    <GrantWrapper
+      pageTitle={formatMessage(m.home.title)}
+      pageDescription={formatMessage(m.home.description)}
+      pageFeaturedImage={formatMessage(m.home.featuredImage)}
```

Comment:
I added a couple of fields to the Custom Page content type in Contentful a while back for this use case.

![Screenshot 2024-11-14 at 09 46 11](https://github.com/user-attachments/assets/89b20257-3c55-4292-85d1-1d3e1267e1cf)

These fields are used in the CustomPageWrapper to set meta tags.

Reply:
So feel free to use them, or not. Just wanted to let you know if you weren't aware of them

Reply:
Oh hey, thats much nicer